renovate.json

```diff
@@ -7,5 +7,20 @@
     "github>argoproj/argo-cd//renovate-presets/fix/disable-all-updates.json5",
     "github>argoproj/argo-cd//renovate-presets/devtool.json5",
     "github>argoproj/argo-cd//renovate-presets/docs.json5"
-  ]
+  ],
+  "enabledManagers": ["helm"],
```

Comment:
I don't think we need enabledManagers at all

Reply:
yes I saw. I am looking for probable solution. Hit and try! You can see [here](https://github.com/afzal442/argo-cd/pulls) after this change